### PR TITLE
Fix the communautary resource modal

### PR DESCRIPTION
**Pull request by noirbizarre**
The communautary resource modal was missing for 2 reasons:
- metadata weren't exposed into json-ld (added to the non standard contributedDitribution to fix that)
- it wasn't handled at all client-side

### CODE REVIEW

File: udata/templates/dataset/resource/list-item.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% set resource_format = resource.format|trim|lower or 'data' %}
 <div id="resource-{{resource.id}}" class="list-group-item"
     data-checkurl="{{ url_for('api.checkurl') }}"
-    @click="showResource('{{resource.id}}', $event)">
+    @click="showResource('{{resource.id}}', $event, {{ resource.owner is defined|tojson }})">
```
Comment (davidbgk) [inline udata/templates/dataset/resource/list-item.html]: I did not know about that syntax, is that standard?

Comment (jdesboeufs) [inline udata/templates/dataset/resource/list-item.html]: Yep: http://flask.pocoo.org/docs/0.12/templating/#standard-filters

Comment (davidbgk) [inline udata/templates/dataset/resource/list-item.html]: I know that but the prior `is defined` returns a boolean?

Comment (davidbgk) [inline udata/templates/dataset/resource/list-item.html]: Oh wait I get it, that what we actually want :)

File: js/front/dataset/index.js [hunk 2]
```diff
@@ -82,21 +79,24 @@ new Vue({
...
             dataset.keywords = dataset.keywords.split(',').map(keyword => keyword.trim());
             return dataset;
         },
 
         /**
-         * Display a resource in a modal
+         * Display a resource or a community ressource in a modal
          */
-        showResource(id, e) {
+        showResource(id, e, community) {
```
Comment (davidbgk) [inline js/front/dataset/index.js]: Can we switch to `isCommunity` for a more explicit naming?

File: udata/templates/dataset/resource/list-item.html (hunk 1 above)
Comment (noirbizarre) [inline udata/templates/dataset/resource/list-item.html]: ;) It's the weird but working mix between jinja and Vue.js

File: js/front/dataset/index.js (hunk 2 above)
Comment (noirbizarre) [inline js/front/dataset/index.js]: done ;)

**Pull request merged**